__tests__/app.test.js

```diff
@@ -32,4 +33,16 @@ describe("*", () => {
             expect(body.message).toBe('path not found');
         })
     })
+})
+
+describe("/api", () => {
+    test("GET 200: responds with an object describing all the available endpoints on the API", () => {
+        return request(app)
+        .get("/api")
+        .expect(200)
+        .then(({res}) => {
```

Comment:
Looks like you are destructuring res off of the response (equivalent to doing response.res)? Try this with the body key instead of res (response.body), you shouldn't need to parse this!